addons/udes_stock/models/stock_move_line.py

```diff
@@ -775,3 +776,24 @@ def any_destination_locations_default(self):
         )
         default_dest.ensure_one()
         return any(ml.location_dest_id == default_dest for ml in self)
+
+    def package_name_policy(self):
```

Comment:
Please rename this method to "new_package_name", since it returns a name (rather than returning something representing a policy).